features/support/simplecov_setup.rb

```diff
@@ -0,0 +1,12 @@
+# @note this file is loaded in env.rb to setup simplecov using RUBYOPTs for child processes
+
+require 'simplecov'
+
+require 'pathname'
+
+root = Pathname(__FILE__).expand_path.parent.parent.parent
+
+SimpleCov.command_name(ENV['SIMPLECOV_COMMAND_NAME'])
+SimpleCov.root(root)
+load root.join('.simplecov')
```

Comment:
This is loading the `.simplecov` configuration file.  Yes, it is required because when `require 'simplecov'` is called, Simplecov will try to load `'.simplecov'`, but the working directory will be wrong (because it will be tmp/aruba), so I have to explicitly set the root first, `SimpleCov.root(root)` and then load the configuration file `load root.join('.simplecov')`